laba2/laba2-1.php

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Laba_2</title>
+    <style type = "text/css">
+        .menu
+        {   
+            width: 100%;
+            padding: 0;
+            margin: 0 auto;
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            list-style: none;
+        }
+        a
+        {
+            padding: 20px;
+            margin: 0;
+            text-decoration: none;
+            text-transform: uppercase;
+            font-weight: 600;
+            font-size: 22px;
+            color: black;
+        }   
+        .currentElement
+        {
+            background-color: gray;
+        }
+    </style>
+    <?php
+        $navigation = array('Main', "Books", "Series", "Contacts");
+        if(isset($_GET["index"]))
+            $index = $_GET["index"];
+        else
+            $index = 0;
+    ?>
+</head>
+<body>
+    <header>
+        <ul class="menu">
+            <?php 
+            foreach($navigation as $key => $value) {
+                ?>
+                <li>
+                    <?php
+                    if ($index == $key) {
+                        ?>
+                        <a class = "currentElement" href="#"><?php echo $value ?></a>
+                        <?php
+                    } else {
+                        ?>
+                        <a href="laba2-1.php?index=<?=$key?>"><?php echo $value ?></a>
```

Comment:
У вас навигация не влияет на содержимое страницы?

Reply:
Да, но думаю, что это можно исправить используя `$_SERVER` для начальной инициализации `index`(при старте работы с разных страниц), и изменить ссылку на что-то такое
```suggestion
                        <a href="laba2-<?=$key?>.php?index=<?=$key?>"><?php echo $value ?></a>
```

Reply:
Именно так, и ?index=... не нужно добавлять, вы можете определить страницу по пути